subsys/bluetooth/host/ecc.h

```diff
@@ -44,6 +44,16 @@ struct bt_pub_key_cb {
  */
 bool bt_pub_key_is_debug(uint8_t *cmp_pub_key);
 
+/*  @brief Check if public key is valid.
+ *
+ *  Verify that the public key is valid, e.g. that its coordinates lie on the eliptic curve.
+ *
+ *  @param key The public key to validate.
+ *
+ *  @return True if the public key is valid.
+ */
+bool bt_pub_key_is_valid(const uint8_t *key);
```

Comment:
Suggest to define the size of the key. I assume it's a static size of 64 octets?